### PR TITLE
amadeus: Update to REV9

### PR DESCRIPTION
**Pull request by Thog**
This implements all the changes made with REV9 on 12.0.0.

### CODE REVIEW

File: Ryujinx.Audio/Renderer/Parameter/Effect/LimiterParameter.cs [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        /// The attack time.
+        /// <remarks>This is in microseconds.</remarks>
+        /// </summary>
+        public int AttackTimeUs;
+
+        /// <summary>
+        /// The release time.
+        /// <remarks>This is in microseconds.</remarks>
+        /// </summary>
+        public int ReleaseTimeUs;
```
Comment (AcK77) [inline Ryujinx.Audio/Renderer/Parameter/Effect/LimiterParameter.cs:72]: Sometimes you specified "Us" in the name, sometimes you don’t even if the comment say it’s in "micro second", maybe you should do the same thing everything for consistency? Imo removing "Us" is better.

**Review by AcK77 — APPROVED** [review]
Just one nit, instead of that it lgtm!

File: Ryujinx.Audio/Renderer/Parameter/Effect/LimiterParameter.cs (hunk 1 above)
Comment (Thog) [reply Ryujinx.Audio/Renderer/Parameter/Effect/LimiterParameter.cs:72]: totally missed to clean that out thanks 👍

**Review by Thealexbarney — APPROVED** [review]
LGTM other than the `Memory<T>.Span` thing we talked about

**Pull request merged**